policyengine/countries/uk/default_reform/reform.py

```diff
@@ -602,8 +598,38 @@ def formula(household, period, parameters):
                 )
                 > 0
             )
+
+    class smf_tax_payment_eligible(Variable):
+        entity = Household
+        definition_period = YEAR
+        label = "Eligible for SMF tax-related payments"
+        value_type = bool
+
+        def formula(household, period):
+            person = household.members
+            tax_band = person("tax_band", period)
+            bands = tax_band.possible_values
+
+            return household.any(
+                (tax_band == bands.BASIC)
+                | (tax_band == bands.STARTER)
+                | (tax_band == bands.INTERMEDIATE)
+            ) * ~household("smf_benefit_payment_eligible", period)
+
+    class smf_benefit_cash_payment(Variable):
+        entity = Household
+        definition_period = YEAR
+        label = "Benefit-based cash payment"
+        documentation = (
+            "The benefit-based cash payment this household receives."
```

Comment:
Similar for the tax one
```suggestion
            "The Social Market Foundation's benefit-based cash payment this household receives."
```